app/registries/foreman/setting_manager.rb

```diff
@@ -37,25 +37,32 @@ def storage
         SettingManager.settings
       end
 
+      def available_types
+        [:boolean, :integer, :float, :string, :text, :hash, :array]
```

Comment:
What's text and how that is different from string?

Reply:
I would imagine the UI part: string is probably short and would translate to an input while text is long (think welcome text) is a text area.

Reply:
Exactly and text should accept markdown input.